### PR TITLE
Add global error handler

### LINKED ISSUE #91: catch errors globally and alway display them
## expectations
- never see a screen like this
Screenshot_20190906-110359__01
- display all errors on screen => allow debugging on mobile without console

### PR DESCRIPTION
**Pull request by vrde**
Fix #91

It looks like this:
signal-attachment-2019-09-06-165515_002

If you open details, you'll see the actual stacktrace... :tada: SOURCEMAPPED :tada:

signal-attachment-2019-09-06-165515_001

I see the sourcemap is served also in production, so everything should work as expected.

The call to action for the error is "let us know" and "copy to clipboard". If you have better ideas for it please tell me.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -867,6 +867,8 @@ class App extends Component {
 
     const { voteStartTime, voteEndTime, trashBox } = this.state;
     const web3Props = { plasma: xdaiweb3, web3, account, metaAccount };
+    throw new Error("🍌");
```
Comment (TimDaub) [inline src/App.js]: remove before merging?

**Comment by troggy** [comment]
maybe log to Sentry like Max proposed?
https://github.com/deora-earth/voting-frontend/pull/80/files#diff-7004115ba56a5a314d592b60c193adefR7
https://github.com/deora-earth/voting-frontend/pull/80/files#diff-1fdf421c05c1140f6d71444ea2b27638R17

File: src/App.js (hunk 1 above)
Comment (vrde) [reply src/App.js]: :sweat_smile:

**Comment by vrde** [comment]
> maybe log to Sentry like Max proposed?

Yeah I can do it. I think Sentry is a better approach than showing the error to the end user. Should I close this PR in favor of #80?

**Comment by troggy** [comment]
> Should I close this PR in favor of #80?

No, still need to show meaningful error message to user (so we can distinguish) + ideally log a stack trace to the server.

**Review by troggy — APPROVED** [review]
🍌

**Review by troggy — APPROVED** [review]

**Pull request merged**